### ISSUE TITLE
i3-next-workspace crashes in Regolith 3.0

### ISSUE THREAD

**Issue by kgilmer** [regolith-linux/regolith-desktop#844]
2023-04-22T10:25:13.513751-07:00 ana regolith.desktop[2724510]: Traceback (most recent call last):
2023-04-22T10:25:13.513846-07:00 ana regolith.desktop[2724510]:   File "/usr/bin/i3-next-workspace", line 38, in <module>
2023-04-22T10:25:13.513875-07:00 ana regolith.desktop[2724510]:     startnum = min(mapping_names.keys())
2023-04-22T10:25:13.513892-07:00 ana regolith.desktop[2724510]:                ^^^^^^^^^^^^^^^^^^^^^^^^^
2023-04-22T10:25:13.513909-07:00 ana regolith.desktop[2724510]: ValueError: min() arg is an empty sequence

**Comment by kgilmer** [comment]
The code expects i3 workspace names to begin with "i3-wm" however we've changed these keys to be wm agnostic for wayland.

**Comment by cfsmp3** [comment]
> The code expects i3 workspace names to begin with "i3-wm" however we've changed these keys to be wm agnostic for wayland.

On a very quick look, the first issue is that `xrdb -query` doesn't return anything.

**Comment by kgilmer** [comment]
Yep, thanks @cfsmp3 !  There is a PR if you'd care to verify but I don't think this is a very complex issue..

**Comment by cfsmp3** [comment]
> Yep, thanks @cfsmp3 ! There is a PR if you'd care to verify but I don't think this is a very complex issue..

Seems easy, but the initial issue I have (note that I'm not totally up-to-date, still 2.2 more or less stable) is that the query comes empty, so the issue happens (for me) before the filter with the names.
